src/main/java/ch/njol/skript/entity/SimpleEntityData.java

```diff
@@ -271,6 +277,16 @@ public boolean equals(final @Nullable Object obj) {
 			types.add(new SimpleEntityDataInfo("trident", Trident.class));
 
 
+		}
+		if (Skript.isRunningMinecraft(1, 14)) {
+			types.add(new SimpleEntityDataInfo("fox", Fox.class));
```

Comment:
1. We may need to discuss whether we will implement Pandas this way or create a custom entity data to spawn them like: `playful panda`, `agressive panda`, etc. Of course, both having expressions for changing their main and hidden gene. Kind of the same for Fox but since Spigot doesn't provide any API for them yet, we'll skip that for now.

2. Llamas definitely needs a custom entity data, so we can spawn colored llamas and now a trader llama too like `red llama`, `blue llama`, `trade llama`.

3. Wandering trader is a villager type so it should belong to the VillagerData class.

4. You forgot the raider Raider supertype here.

Reply:
1) Right, in my head I talked to you about this, but I totally didn't. I was going to ask if you could take care of that, like when you did the fish back in 1.13?
Should I leave it for now, so at least we have the pandas in the first update, and then tend to a panda class later?

2) Ahh yeah, that would be a good idea. Kind of the same thing with pandas? I should leave this here for now?

3) I totally thought they were a separate entity. Considering theres one villager entity, with professions, and this guy is all on his lonesome. Im not really sure what to do here?

4) My bad, Ill add that!